20150310-wp-core-contrib-day.md

```diff
@@ -0,0 +1,255 @@
+# WP Core Contrib Day
+
+![WP Core Contrib Day](https://s3.amazonaws.com/cdn.deliciousbrains.com/content/uploads/2015/03/Make-WordPress-Core.png)
+
+Every month the staff here at Delicious Brains take a "WP Core Contrib Day", a day to give back to [WordPress](https://wordpress.org) Core. This is an important day for us as we make our living from premium WordPress plugins, using our skills to help the open source WordPress Core keep on being awesome just feels right.
```

Comment:
> Is it worth mentioning Matt M's 5 for the future here? http://ma.tt/2014/09/five-for-the-future/

Good idea. I was considering doing a company contrib day before Matt's post, but it definitely pushed me to make it a reality.